lib/util/http/github.ts

```diff
@@ -223,14 +223,18 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
     return result;
   }
 
-  private async getGraphql<T = unknown>(
+  public async getGraphql<T = unknown>(
     query: string,
-    accept = 'application/vnd.github.merge-info-preview+json'
-  ): Promise<GithubGraphqlResponse<T>> {
+    options: GraphqlOptions = {}
+  ): Promise<T> {
     let result = null;
 
     const path = 'graphql';
 
+    const {
+      acceptHeader: accept = 'application/vnd.github.merge-info-preview+json',
```

Comment:
This is only necessary for PRs I think. Do we send it with every request? https://developer.github.com/v4/previews/#mergeinfopreview---more-detailed-information-about-a-pull-requests-merge-state

Reply:
It was default parameter when this function was private, need to see how much snapshots use it

Reply:
I think ideally it should be provide by the calling function and not added by default. It's only used to enable a certain field in Pull Requests

Reply:
So for the entire Github HTTP, the default one is `application/vnd.github.v3+json`, but for app mode it's `application/vnd.github.merge-info-preview+json, application/vnd.github.v3+json`? And never `application/json` except being set explicitly?

Reply:
I think that's correct

Reply:
Well, that's pretty tricky to make it consistent with current snapshots, so take a look if `accept`-related changes are good

Reply:
It seems you can send only one header (the most specific/detailed) but I prefer to send 2 for future compatibility